WebHostLib/static/assets/player-settings.js

```diff
@@ -338,7 +430,21 @@ const updateGameSetting = (settingElement) => {
 
 const exportSettings = () => {
   const settings = JSON.parse(localStorage.getItem(gameName));
-  if (!settings.name || settings.name.toLowerCase() === 'player' || settings.name.trim().length === 0) {
+  const preset = localStorage.getItem(`${gameName}-preset`);
+  switch (preset) {
+    case "__default":
+      settings["description"] = `Generated by https://archipelago.gg with the default preset.`;
```

Comment:
This case does exactly the same thing as the default case, it's therefore not necessary

Reply:
Negative, because it would output `Generated by https://archipelago.gg with the __default preset.` because it uses the value (not the display name).

Reply:
And I don't want that :P

Reply:
In that case, if __default and __custom include the leading underscores, would other presets also include it? Should it maybe get trimmed away here?

Reply:
Other preset names shouldn't have the leading underscores.